recipes/default.rb

```diff
@@ -16,14 +16,34 @@
 # See the License for the specific language governing permissions and
 # limitations under the License.
 
+pg = Chef::EncryptedDataBagItem.load(node['timesync']['databag'],
+                                     'pg')
+
+environment = {
+  'PG_CONNECTION_STRING' => "postgres://#{pg['user']}:#{pg['pass']}@" \
+    "#{pg['host']}:#{pg['port']}/#{pg['database_name']}",
+  'NODE_ENV' => 'production'
+}
+
 nodejs_webapp 'timesync' do
   path node['timesync']['application_path']
-  create_user true
   user node['timesync']['user']
   group node['timesync']['group']
+  env environment
 
   script 'src/app.js'
   repository node['timesync']['repo']
   branch node['timesync']['branch']
   node_args node['timesync']['node_args']
 end
+
+bash 'run timesync migrations' do
+  code 'npm run migrations'
+  env environment
+  cwd "#{node['timesync']['application_path']}/source"
+end
```

Comment:
If the migrations have already been run, nothing will be changed. Otherwise, the database and the app need to be updated at the same time. It's not the **best** way to go about running migrations, but for a single machine setup I think it works.